cloud/clouds.go

```diff
@@ -395,12 +398,59 @@ func ParseOneCloud(data []byte) (Cloud, error) {
 }
 
 // ParseCloudMetadata parses the given yaml bytes into Clouds metadata.
+//
+// The expected regular yaml formal is:
+//
+// clouds:
+//   garage-maas:
+//     type: maas
+//     auth-types: [oauth1]
+//     endpoint: "http://garagemaas"
+//     skip-tls-verify: true`
+//   ...
+//
+// It also accepts a yaml format without the 'clouds' key at the top,
+// e.g.
+//
+// garage-maas:
+//   type: maas
+//   auth-types: [oauth1]
+//   endpoint: "http://garagemaas"
+//   skip-tls-verify: true`
+// ...
+//
 func ParseCloudMetadata(data []byte) (map[string]Cloud, error) {
 	var metadata cloudSet
-	if err := yaml.Unmarshal(data, &metadata); err != nil {
+
+	// Unmarshal with a generic type first
+	yaml_map := make(map[string]interface{})
```

Comment:
As this is golang, variables are camel case and not snake case

Reply:
> As this is golang, variables are camel case and not snake case

Oops, adapting to new styles is hard, man! :)